{
  "name": "fs-exists",
  "description": "Wraps fs.exists so callback is called with (err, result) instead of just (result).",
  "keywords": [],
  "version": "0.1.0",
  "author": "Meryn Stol <merynstol@gmail.com>",
  "homepage": "https://github.com/meryn/fs-exists",
  "repository": {
    "type": "git",
    "url": "git://github.com/meryn/fs-exists.git"
  },
  "private": false,
  "dependencies": {},
  "devDependencies": {
    "coffee-script": "~1.12.4",
    "async": "~2.1.5",
    "mocha": "~3.2.0"
  },
  "optionalDependencies": {},
  "engines": {
    "node": "0.10.x",
    "npm": "1.2.x"
  },
  "main": "lib/fs-exists.js",
  "scripts": {
    "prepublish": "npm test",
    "pretest": "make build",
    "test": "make test"
  }
}
